google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -103,6 +103,9 @@
     <component>
       <implementation-class>com.google.cloud.tools.intellij.startup.CloudToolsPluginInitializationComponent</implementation-class>
     </component>
+    <component>
+      <implementation-class>com.google.cloud.tools.intellij.startup.ConflictingAppEnginePluginCheck</implementation-class>
```

Comment:
It certainly can. I was trying to avoid bloating that class too much, but I'm happy to call the ConflictingAppEnginePluginCheck from there.